options/configurable.cc

```diff
@@ -608,7 +608,8 @@ Status ConfigurableHelper::SerializeOptions(const ConfigOptions& config_options,
           }
           if (!s.ok()) {
             return s;
-          } else if (!value.empty()) {
+          } else if (!value.empty() ||
+                     (opt_info.IsVector() && opt_info.IsMutable())) {
```

Comment:
Not sure what this change has to do with the rest of this PR?

Reply:
This has to do with making the CircleCI complete.  There is a test that randomizes option values and then attempts to serialize and parse.  It started putting zero has number of levels, num_levels, consistently.  That demonstrated that a zero length vector did not work.

My goal was to simply get the test to pass.  The alternate fix is to change the random number for num_levels from 0 through 99 to be 1 through 100.  But again it has nothing to do with my change ... like some of the white space only files.